doc-Deployment_Planning_Guide/topics/capacity.adoc

```diff
@@ -55,94 +55,116 @@ After creating the new user, add the user's credentials to the settings for the
 . Click *Save*.
 . Restart the Capacity and Utilization Data Collector.
 
-[[data_collection_for_rhev_33_34]]
-=== Data Collection for Red Hat Enterprise Virtualization
+[[data_collection_for_rhev_4.x]]
+=== Data Collection for Red Hat Virtualization
 
-[NOTE]
-====
-The procedure applies to version numbers Red Hat Enterprise Virtualization 3.x and Red Hat Virtualization 4.x. 
-====
-
-To collect capacity and utilization data for Red Hat Enterprise
-Virtualization, you must add {product-title} as
-a user to the RHEV-M database.
+To collect capacity and utilization data for Red Hat Virtualization (RHV), you must add {product-title} as
+a user to the RHV-M database.
 
 Perform this procedure on the PostgreSQL server where the history
-database is located. Usually, this is the RHEV-M server.
+database (ovirt_engine_history) is located. Usually, this is the RHV-M server.
 
-. Using SSH, access the RHEV-M database server as the root user:
+. Using SSH, access the RHV-M database server as the root user:
 +
-------
+----
 $ ssh root@example.postgres.server
-------
+----
 +
 . Switch to the postgres user:
 +
-------
+----
 # su - postgres
-------
+----
++
+[NOTE]
+====
+For RHV 4.2, the PostgreSQL database is delivered as a software collection in version 9.5 and must be enabled first. Therefore, in the following steps, to run psql commands you will need to prefix each command with: *_scl enable rh-postgresql95 --_* <psql command>
```

Comment:
Great, done! Added the instruction to source the enable script. Earlier, I tried to avoid prefixing the scl enable rh-postgresql95 -- a little differently but ran into some issues at the grant permission step when testing for 4.2. So this works well. Thanks for your feedback and suggestions, Brant!